rust/matches/src/utils.rs

```diff
@@ -19,7 +19,9 @@ use anchor_spl::token::{Mint, Token};
 use arrayref::array_ref;
 use spl_associated_token_account::get_associated_token_address;
 use spl_token::instruction::close_account;
-use std::convert::TryInto;
+use std::{convert::TryInto, str::FromStr};
+
+pub const NAMESPACE_ID: &str = "nameAxQRRBnd4kLfsVoZBBXfrByZdZTkh8mULLxLyqV";
```

Comment:
Man, I really wish all these IDs around the raindrops code could move into one global constants.rs file.

Reply:
+1, we need a commons crate